packages/rps/src/utils/channel-client.ts

```diff
@@ -61,7 +61,7 @@ type UnsubscribeFunction = () => void;
 // The message Payload is designed to be opaque to the app. However, it's useful
 // to be able to specify the Payload type for the FakeChannelClient, as we'll be
 // manipulating it within the client.
-export interface IChannelClient<Payload = any> {
+export interface ChannelClient<Payload = any> {

```

Comment:
Why was this necessary?  I would have thought the prefixed `I` was pretty helpful in distinguishing this interface from some class that implements it.

Reply:
I see you resolved this :)